samples-js/functions/weatherMonitor.js

```diff
@@ -0,0 +1,129 @@
+const { output } = require("@azure/functions");
+const df = require("durable-functions");
+const { DateTime } = require("luxon");
+const axios = require("axios").default;
+
+const clearWeatherConditions = ["clear sky", "few clouds", "scattered clouds", "broken clouds"];
+
+const getIsClearActivity = "getIsClear";
+const sendGoodWeatherAlertActivity = "sendGoodWeatherAlert";
```

Comment:
Just noting that I'm not a big fan of these constant strings across the file, which feel like a boilerplate. I would love it if we could have these templates benefit from any of the proposals [here](https://github.com/Azure/azure-functions-durable-js/issues/418) asap.

Reply:
I was planning on updating the samples once we have a conclusion and implementation on #418. Another issue I have with these string constants is that they don't work when the functions are registered in different files, unless you export and import them, which is more hassle than it might be worth. For the meantime, do you prefer keeping the constants or just having the plain strings?

Reply:
Let's keep the constants for now :)